wiki/Skinning/Compendium/en.md

```diff
@@ -0,0 +1,269 @@
+[o!s]: /wiki/shared/mode/osu.png "osu!standard"
+[o!t]: /wiki/shared/mode/taiko.png "osu!taiko"
+[o!c]: /wiki/shared/mode/catch.png "osu!catch"
+[o!m]: /wiki/shared/mode/mania.png "osu!mania"
+
+# Skins Compendium
+This is a compendium of all completed osu! skins
+
+- ![][o!s] This means the skins contains osu!standard elements
```

Comment:
remove `This` at the front of each sentence and add a full stop to the end of each line